dash/dash-config.cmake.in

```diff
@@ -8,14 +8,14 @@
 
 @PACKAGE_INIT@
 
-set(DASH_VERSION_MAJOR "@DASH_VERSION_MAJOR@")
-set(DASH_VERSION_MINOR "@DASH_VERSION_MINOR@")
-set(DASH_VERSION_PATCH "@DASH_VERSION_PATCH@")
+set(DASH_VERSION_MAJOR "@DASH_VERSION_MAJOR@")
+set(DASH_VERSION_MINOR "@DASH_VERSION_MINOR@")
+set(DASH_VERSION_PATCH "@DASH_VERSION_PATCH@")
 
 set(DASH_LIBRARY "@DASH_LIBRARY@")
 set(DASH_LIBRARIES ${DASH_LIBRARY} "dart-@dart_variant@")
 set(DASH_CXX_FLAGS "@VARIANT_ADDITIONAL_COMPILE_FLAGS@ @CMAKE_CXX_FLAGS_RELEASE@")
```

Comment:
[This is getting offtopic for this PR]

Ok, I must have been running in circles, just using `target_link_libraries()` with `${DASH_LIBRARIES}` is sufficient. But do I need to use the CXX flags from DASH too?